docs/getting_started/basic/model_eval/step1.ipynb

```diff
@@ -0,0 +1,995 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">In our first step, we will be covering how we can perform basic model evaluation. One of the great parts of FiftyOne is that once your data and model predictions are imported into FiftyOne, evaluation becomes easy. Gone are the days of converting your YOLO styled predictions to COCO styled evaluation, FiftyOne becomes the central location view, manage, and evaluate your data AND you model output. FiftyOne handles all the conversions for you so you can focus on the task at hand.
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/thesteve0/v51-gs-draft/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>